easybuild/easyconfigs/n/NVHPC/NVHPC-22.1-CUDA-11.4.1.eb

```diff
@@ -0,0 +1,58 @@
+name = 'NVHPC'
+version = '22.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+local_tarball_tmpl = 'nvhpc_2022_%%(version_major)s%%(version_minor)s_Linux_%s_cuda_multi.tar.gz'
+# By downloading, you accept the HPC SDK Software License Agreement
+# https://docs.nvidia.com/hpc-sdk/eula/index.html
+# accept_eula = True
+source_urls = ['https://developer.download.nvidia.com/hpc-sdk/%(version)s/']
+sources = [local_tarball_tmpl % '%(arch)s']
+checksums = ['7e4366509ed9031ff271e73327dd3121909902a81ac436307801a5373efaff5e']
+
+
+local_gccver = '11.2.0'
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.37', '', ('GCCcore', local_gccver)),
+    # This is necessary to avoid cases where just libnuma.so.1 is present in the system and -lnuma fails
+    ('numactl', '2.0.14', '', ('GCCcore', local_gccver)),
+    ('CUDA', '11.4.1', '', True),
```

Comment:
@mustafabar is using an external CUDA v11.4 supported by NVHPC? I only see versions 10.2, 11.0  and 11.5 of CUDA in `%(installdir)s/Linux_x86_64/22.1/cuda/`.

Reply:
@lexming it worked with the easybuild 11.4.1 installed version. It also comes with the bundled versions you mentioned.  I could try with the bundled version and only use the existing `default_cuda_version` variable to be consistent with older versions.

Reply:
@lexming but I think it is more convenient to use the EB CUDA. Libraries that rely on NVHPC may also need CUDA for clear reasons (e.g. the GPU version of QuantumESPRESSO), and loading EB CUDA with the NVHPC makes life easier.

Reply:
@mustafabar thanks for the explanation, if it works and is useful to you then seems fine :+1: